metrics/src/main/java/io/opentelemetry/example/metrics/HistogramExample.java

```diff
@@ -0,0 +1,45 @@
+package io.opentelemetry.example.metrics;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.Meter;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class HistogramExample {
+
+  private final OpenTelemetry otel;
+
+  public HistogramExample(OpenTelemetry otel) {
+    this.otel = otel;
+  }
+
+  private void run() {
+    Meter meter = otel.getMeter("io.opentelemetry.example.metrics");
+
+    LongHistogram histogram =
+        meter
+            .histogramBuilder("people.ages")
+            .setDescription("A distribution of people's ages")
+            .setUnit("years")
+            .ofLongs()
+            .build();
+
+    Random rand = new Random();
+    Attributes attrs =
+        Attributes.of(
+            stringKey("decade"), "1990s",
+            stringKey("source"), "someAlmanac");
+    IntStream.range(0, 1000).forEach(x -> histogram.record(rand.nextLong(115), attrs));
+  }
+
+  public static void main(String[] args) {
+    System.setProperty("otel.metric.export.interval", "10000");
+    OpenTelemetry sdk = GlobalOpenTelemetry.get();
```

Comment:
Nit: this will only return a real `OpenTelemetry` when `otel.java.global-autoconfigure.enabled` is set to true. I've noticed that other examples also use the global; should we perhaps replace it with `AutoConfiguredOpenTelemetrySdk`?

Reply:
Yeah, I agree. Created #155 to track.

Reply:
Thanks!